### ISSUE TITLE
Last version still crashes Nuget Package Manager for VS2015 

### ISSUE THREAD

**Issue by FredyFLU** [rebuy-de/rb-forms-barcode#57]
When using Nuget Packet Manager for Visual Studio 2015 v3.4.4.13.21 (last update on 07/10/2016), if the package (0.7.0.1) is added to any project, this crashes the manager : 

Added package 'Rb.Forms.Barcode.0.7.1' to 'packages.config'
Successfully installed 'Rb.Forms.Barcode 0.7.1' to GeoWorker.BarcodeScanner
========== Finished ==========
**ZIP archive contains unsupported data structures.**

**Comment by calbert52** [comment]
Same here, even worse, selecting 'Installed' in 'Manage Packages for Solution' displays "Error occurred" and none of your installed packages are displayed. No unable to uninstall installed packages.

**Comment by fvermeulen** [comment]
@FredyFLU @calbert52 you are both on windows right? We are developing on mac. If that is true, than we have to check the path structure and rebuild the library.

Are you completely new to this barcode scanner, or do you already had an older version from the library installed?

**Comment by calbert52** [comment]
Yes, I am using windows. This is the first time I tried using it in a project.

Thanks

On Nov 24, 2016, at 9:02 AM, Fokke Vermeulen <notifications@github.com<mailto:notifications@github.com>> wrote:


@FredyFLU<https://github.com/FredyFLU> @calbert52<https://github.com/calbert52> you are both on windows right? We are developing on mac. If that is true, than we have to check the path structure and rebuild the library.

Are you completely new to this barcode scanner, or do you already had an older version from the library installed?

-

**Comment by FredyFLU** [comment]
Yes I am also using Windows with Visual Studio 2015 Update 3.
I used and older version of the barcode scanner without problem but this may be related to the package manager that was updated recently.
Thanks for help.

**Comment by marcusts** [comment]
This is also happening to me on a Windows 10 PC with Visual Studio 2015.  I can only remove rb.forms manually, which is a major hassle.

Temporarily, I can compile your source separately and include the DLL's.  But that is not compliant with my client's corporate coding philosophy.

Please repair immediately, thanks!

**Comment by rdoubleui** [comment]
As @fvermeulen indicated maybe this is related to the file paths. One simple test you could try is to turn around slashes in the `.nuspec` [file references](https://github.com/rebuy-de/rb-forms-barcode/blob/master/Rb.Forms.Barcode.nuspec#L47). Also make sure the relative paths are correct. If that doesn't work let's investigate further from there.

**Comment by jcmanke** [comment]
Turning around the slashes fixed the issue for me. I have opened pull request #67 with the change.
